src/resmoke/external_cmd.rs

```diff
@@ -0,0 +1,40 @@
+use std::process::{Command, Stdio};
+
+use anyhow::{bail, Result};
+use tracing::{event, Level};
+
+/// Run an external command and return the output.
+///
+/// # Arguments
+///
+/// * `command` - Command with arguments to run.
+///
+/// # Return
+///
+/// The output of the command.
+pub fn run_command(command: &[&str]) -> Result<String> {
+    let binary = command[0];
+    let args = &command[1..];
+    let cmd = Command::new(binary)
+        .args(args)
+        .stdout(Stdio::piped())
+        .stderr(Stdio::piped())
+        .spawn()?
+        .wait_with_output()?;
+
+    if !cmd.status.success() {
+        let error_message = String::from_utf8_lossy(&cmd.stderr).to_string();
+
+        event!(
```

Comment:
With this function we will log stderr output when we encounter an error trying to execute commands. We should be able to switch resmoke calls over to use this function and then they will have better error messages. I didn't add this here to keep this PR focused.